src/views/Accessibility/AccessibilityRequestDetailPageRTL.test.tsx

```diff
@@ -162,5 +162,161 @@ describe('AccessibilityRequestDetailPage', () => {
       const [, notesList] = getAllByRole('list');
       expect(within(notesList).getAllByRole('listitem').length).toEqual(2);
     });
+
+    describe('Add a note', () => {
+      it('shows a success message', async () => {
+        const createNoteMocks = [
+          testerMocks[0],
```

Comment:
If you want to re-use this mock, store the mock object in its own variable and name it appropriately.

Since the query data doesn't matter, I think it would be best to create a mock query with the bare minimum/default data.